server/routes/recentArrivals/recentArrivalsController.test.ts

```diff
@@ -67,3 +67,16 @@ describe('GET /recent-arrivals', () => {
       })
   })
 })
+
+describe('POST /recent-arrivals', () => {
+  const flash = flashProvider.mockReturnValue([])
+  it('should store search query in flash and redirect to /recent-arrivals/search', () => {
+    return request(app)
+      .post('/recent-arrivals')
+      .expect(302)
+      .expect('Location', '/recent-arrivals/search')
+      .expect(res => {
+        expect(flash).toBeCalledTimes(1)
+      })
```

Comment:
sounds like this test should demo sending the search query and should show what flash is being called with